cellarium/cas/client.py

```diff
@@ -229,14 +238,21 @@ def annotate_anndata_file(
         feature_ids_column_name: str = "index",
     ) -> t.List[t.Dict[str, t.Any]]:
         """
-        Read `anndata.AnnData` matrix and apply the annotate method on it.
+        Read the 'h5ad' file into a :class:anndata.AnnData matrix and apply the :meth:annotate method to it.
 
-        :param filepath: Filepath of the local `anndata.AnnData` matrix
+        :param filepath: Filepath of the local :class:`anndata.AnnData` matrix
         :param chunk_size: Size of chunks to split on
-        :param feature_schema_name: feature schema name to use for data preparation. if 'default' default schema will
-            be used.
-        :param count_matrix_name:  Where to obtain a feature expression count matrix from. Choice of: 'X', 'raw.X'
-        :param feature_ids_column_name: Column name where to obtain Ensembl feature ids. Default `index`.
+        :param feature_schema_name: feature schema name to use for data preparation. |br|
```

Comment:
Make sure that descriptions are the same for matching arguments in different `annotate_*` methods